wp-content/plugins/core/src/Routes/Routes_Subscriber.php

```diff
@@ -1,11 +1,5 @@
 <?php declare(strict_types=1);
 
-/**
```

Comment:
Why remove contextual comments?

Reply:
It was duplicated notes from the method, so just providing extra noise: https://github.com/moderntribe/square-one/blob/ba44b39ecabce2fc4c46be4b99187e55b8644c62/wp-content/plugins/core/src/Routes/Routes_Definer.php#L14-L18